src/utils.js

```diff
@@ -151,7 +151,7 @@ export function getCSSCustomProperty(element, propertyName) {
  * @param {string|number} propertyValue The property value.
  */
 export function setCSSCustomProperty(element, propertyName, propertyValue) {
-  element.style.setProperty(propertyName, propertyValue);
+  element.style.setProperty(propertyName, `${propertyValue}`);
```

Comment:
Any particular reason to force it to be a string?

Reply:
setProperty expects a string https://developer.mozilla.org/en-US/docs/Web/API/CSSStyleDeclaration/setProperty